.github/workflows/continuous-integration.yaml

```diff
@@ -589,54 +586,86 @@ jobs:
             ${{
               steps.commit-conformance.outcome == 'success' 
               && steps.make-pcc.outcome == 'success' 
-              && steps.build-docs.outcome != 'failure'
               && steps.determinism.outcome != 'failure'
+              && steps.build-docs.outcome != 'failure'
+              && steps.check_links.outcome != 'failure'
             }}
         run: |
           if [[ "${CONFORMANCE_STATUS}" != "true" ]]; then
             echo "Conformance failed, got:"
             echo "Commit conformance success step: ${{ steps.commit-conformance.outcome }}"
             echo "Make conformance step: ${{ steps.make-pcc.outcome }}"
-            echo "Build docs step: ${{ steps.build-docs.outcome }}"
             echo "Determinism step: ${{ steps.determinism.outcome }}"
+            echo "Build docs step: ${{ steps.build-docs.outcome }}"
+            echo "Check links step: ${{ steps.check_links.outcome }}"
             exit 1
           fi
 
+      # Tar the docs for releases with the reference build only
       # Taring the docs allows for much faster upload speed (from ~3min worst case to ~2s best case)
       - name: Tar docs artifacts
```

Comment:
the following is basically avoiding doc management needed for the release when we are not running the CI for a release on the reference build